### PR TITLE
final changes to #1523

### LINKED ISSUE #1523: Display the Provenance Change label with the Note under See More
Step 6 to complete https://github.com/pulibrary/pdc_describe/issues/1336

Create a renderer by inserting the following [after](https://github.com/pulibrary/pdc_describe/blob/main/app/models/work_activity.rb#L284)
```  
class ProvenanceNote < Renderer
    # Returns the message formatted to display provenance note
    def body_html
      message_json = JSON.parse(@work_activity.message)
      change_label = message_json[:change_label] || "Other"
      note =   message_json[:note]
      "<details class='message-html'><summary>#{change_label}</summary>#{note}</details>"
    end
end
```

Make the render get call by adding [before](https://github.com/pulibrary/pdc_describe/blob/main/app/models/work_activity.rb#L104):
```
elsif activity_type == PROVENANCE_NOTES
    ProvenanceNote
```

Update to test created in [Step 3](https://github.com/pulibrary/pdc_describe/issues/1520) 
to add a Provenance note by filling in the form (use [fill_in](https://github.com/pulibrary/pdc_describe/blob/a95803c167c0b9f2f032724af6f699c3ba548f6c/spec/system/work_spec.rb#L16) and [select](https://github.com/pulibrary/pdc_describe/blob/a95803c167c0b9f2f032724af6f699c3ba548f6c/spec/system/pppl_work_create_spec.rb#L38)) and submitting the form ([click_on](https://github.com/pulibrary/pdc_describe/blob/a95803c167c0b9f2f032724af6f699c3ba548f6c/spec/system/work_spec.rb#L19)) and make sure the page includes the change label

### PR DESCRIPTION
**Pull request by FavourNwogbo21**
fixes #1523

### CODE REVIEW

File: app/models/work_activity.rb [hunk 1]
```diff
@@ -298,8 +298,9 @@ def body_html
       change_label = message_hash["change_label"]
       change_label ||= "Change"
       # TODO: Make this show the change label with the note under see changes
-      # See an example on line 171 of this file or on github: https://github.com/pulibrary/pdc_describe/blob/3578f734dcee092c15a9ff1fa778f20a846b47f3/app/models/work_activity.rb#L171
-      "#{change_label}: #{message}"
+      # See an example on line 173, message goes in mapped.join, of this file or on github: https://github.com/pulibrary/pdc_describe/blob/3578f734dcee092c15a9ff1fa778f20a846b47f3/app/models/work_activity.rb#L171
```
Comment (carolyncole) [inline app/models/work_activity.rb:301]: I believe this should be removed
```suggestion
```

File: app/models/work_activity.rb [hunk 2]
```diff
@@ -298,8 +298,9 @@ def body_html
...
+      "<details class='message-html'><summary class='show-changes'>#{change_label}</summary>#{message}</details>"
+      #"#{change_label}: #{message}"
```
Comment (carolyncole) [inline app/models/work_activity.rb:303]: Please remove
```suggestion
```

File: spec/system/provenance_note_spec.rb [hunk 3]
```diff
@@ -29,7 +29,10 @@
       click_on "Add Provenance Note"
       within ".beads" do
         expect(page).to have_content("file_audit")
+        expect(page).not_to have_content("test note")
+        page.find(:css, "summary.show-changes").click
         expect(page).to have_content("test note")
+        #To-do
```
Comment (carolyncole) [inline spec/system/provenance_note_spec.rb:35]: Also Remove
```suggestion
```

**Review by carolyncole — CHANGES_REQUESTED** [review]
@FavourNwogbo21 Just a little bit of clean up is still needed.

Looks good otherwise!

File: app/models/work_activity.rb (hunk 1 above)
Comment (FavourNwogbo21) [reply app/models/work_activity.rb]: Hi Carolyn, I have just made these changes. Thanks

File: spec/system/provenance_note_spec.rb (hunk 3 above)
Comment (FavourNwogbo21) [reply spec/system/provenance_note_spec.rb]: I have just made these changes. Thanks

**Review by carolyncole — APPROVED** [review]

**Pull request merged**